src/main/java/com/somemore/community/service/comment/CreateCommunityCommentService.java

```diff
@@ -0,0 +1,35 @@
+package com.somemore.community.service.comment;
+
+import com.somemore.community.domain.CommunityComment;
+import com.somemore.community.dto.request.CommunityCommentCreateRequestDto;
+import com.somemore.community.repository.comment.CommunityCommentRepository;
+import com.somemore.community.usecase.comment.CreateCommunityCommentUseCase;
+import com.somemore.global.exception.BadRequestException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.UUID;
+
+import static com.somemore.global.exception.ExceptionMessage.NOT_EXISTS_COMMUNITY_COMMENT;
+
+@RequiredArgsConstructor
+@Transactional
+@Service
+public class CreateCommunityCommentService implements CreateCommunityCommentUseCase {
+
+    private final CommunityCommentRepository communityCommentRepository;
+
+    @Override
+    public Long createCommunityComment(CommunityCommentCreateRequestDto requestDto, UUID writerId) {
+
+        CommunityComment communityComment = requestDto.toEntity(writerId);
+
+        if (communityComment.getParentCommentId() != null) {
+            communityCommentRepository.findById(communityComment.getParentCommentId())
+                    .orElseThrow(() -> new BadRequestException(NOT_EXISTS_COMMUNITY_COMMENT.getMessage()));
+        }
```

Comment:
추상화된 private 메서드로 추출할 수 있을 것 같습니다.
```suggestion
    checkParentComment(communityComment.getParentCommentId());
```

Reply:
수정했습니다!